docs/source/conf.py

```diff
@@ -29,7 +29,6 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.graphviz',
-    'sphinx.ext.autosectionlabel'
```

Comment:
Is there a reason you deleted this, I use it for some of the auto links in order to link to some of the section headers?

Reply:
Ok, following on from our discussion that this is because the plugin isn't available on the Diamond version of sphinx then I'll go through and update my links.